### PR TITLE
update ci-label gha for dependabot skip changelog

### PR DESCRIPTION
**Pull request by bjlittle**
### Description

This pull-request updates the labeller to automatically add the `skip changelog` label to `dependabot` generated pull-requests.

---

### CODE REVIEW

File: .github/workflows/ci-label.yml [hunk 1]
```diff
@@ -55,6 +55,6 @@ jobs:
           labels: 'type: performance'
 
       - uses: actions-ecosystem/action-add-labels@v1
-        if: startsWith(github.event.pull_request.head.ref, 'pre-commit-ci-update-config') || startsWith(github.event.pull_request.head.ref, 'conda-lock-auto-update')
+        if: startsWith(github.event.pull_request.head.ref, 'pre-commit-ci-update-config') || startsWith(github.event.pull_request.head.ref, 'conda-lock-auto-update') || startsWith(github.event.pull_request.head.ref, 'dependabot/github_actions/actions')
```
Comment (tkoyama010) [inline .github/workflows/ci-label.yml:58]: It is better to use `github.actor` in my mind. See https://github.com/pyvista/pyvista/blob/6d1583bb51aecea964bde64107d8b2641c1f456b/.github/workflows/label.yml#L29 .

**Review by tkoyama010 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-label.yml (hunk 1 above)
Comment (bjlittle) [reply .github/workflows/ci-label.yml:58]: Neat! Thanks @tkoyama010 :rocket:

**Comment by bjlittle** [comment]
@tkoyama010 Are you happy to approve the PR?

**Review by tkoyama010 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**